packages/hydrogen/src/customer/customer.ts

```diff
@@ -454,20 +515,22 @@ export function createCustomerAccountClient({
       session.set(CUSTOMER_ACCOUNT_SESSION_KEY, {
         accessToken: customerAccessToken,
         expiresAt:
-          new Date(
-            new Date().getTime() + (expires_in! - 120) * 1000,
-          ).getTime() + '',
+          new Date(new Date().getTime() + (expires_in - 120) * 1000).getTime() +
+          '',
         refreshToken: refresh_token,
         idToken: id_token,
-        redirectPath: undefined,
       });
 
+      await exchangeForStorefrontCustomerAccessToken();
+
```

Comment:
realizing we probably want to add some test for this

Reply:
I can add some tests next week 👍